build_targets

```diff
@@ -8,18 +8,21 @@ cheeseburger userdebug yes [recovery.img]
 clover userdebug yes [recovery.img]
 comet userdebug yes [boot.img, dtbo.img, init_boot.img, vendor_boot.img, vendor_kernel_boot.img]
 courbet user yes [recovery.img]
+cheetah userdebug yes [boot.img, dtbo.img, init_boot.img, vendor_boot.img, vendor_kernel_boot.img]
 dubai userdebug no [boot.img, dtbo.img, vendor_boot.img]
 dumpling userdebug yes [recovery.img]
 guacamole userdebug yes [boot.img]
 kiev userdebug yes [recovery.img]
 komodo userdebug yes [boot.img, dtbo.img, init_boot.img, vendor_boot.img, vendor_kernel_boot.img]
 lemonade userdebug yes [boot.img, dtbo.img, vbmeta.img, vendor_boot.img]
 lemonadep userdebug yes [boot.img, dtbo.img, vbmeta.img, vendor_boot.img]
+lynx userdebug yes [boot.img, dtbo.img, init_boot.img, vendor_boot.img, vendor_kernel_boot.img]
 m52xq userdebug yes [recovery.img]
 mars userdebug yes [boot.img, dtbo.img, vendor_boot.img]
 mayfly userdebug yes [boot.img, dtbo.img, vendor_boot.img]
 nx_tab userdebug yes [android.ini, boot.img, bootlogo_android.bmp, icon_android_hue.bmp, recovery.img, install/nx-plat.dtimg, bl31.bin, bl33.bin, boot.scr]
 PL2 userdebug yes [boot.img]
+panther userdebug yes [boot.img, dtbo.img, init_boot.img, vendor_boot.img, vendor_kernel_boot.img]
```

Comment:
needs android_device_panther

Reply:
I understand that 
But python script to pick dt stuff does a search for android_device_codename using GitHub API and that is missing for those
Try and delete local manifest and do an init locally 
You'll see what I mean